content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
 ---
-title: Create an Influx Template
+title: Create an Influx template
 description: >
-  placeholder
+  Use the InfluxDB UI and the `influx pkg export` command to create Influx templates.
 menu:
   v2_0:
     parent: Influx templates
     name: Create a template
-    identifier: Create an Influx Template
+    identifier: Create an Influx template
 weight: 101
 v2.0/tags: [influx templates]
 ---
 
-## Placeholder outline
-- Create a template
-  - `influx pkg`
-  - Package everything vs package specific things
-  - Include secrets (only keys are included, uses will still need to populate the secret)
-  - Submit a template to the InfluxDB community template repo
-  - View your package summary
-  - Validate a package
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create Influx templates.
+Add buckets, Telegraf configurations, tasks, and more in the InfluxDB UI and export
```

Comment:
Add resources (buckets,...and more)...and then save the resources as a template?